generators/src/new-test-model/templates/package/README.md

```diff
@@ -0,0 +1,31 @@
+# @commercetools-test-data/{{modelCodename}}
+
+This package provides the data model for the commercetools platform `{{modelName}}` representations
+
+// TODO: Check this autogenerated docs URL exists and fix if it if it doesn't
+https://docs.commercetools.com/api/projects/{{modelName}}#representations
+
+# Install
+
+```bash
+$ pnpm add -D @commercetools-test-data/{{modelCodename}}
+```
+
+# Usage
+
+```ts
+import {
+  {{modelName}}Rest,
+  {{modelName}}Graphql,
+} from '@commercetools-test-data/{{modelCodename}}';
+
+const rest{{modelName}} = {{modelName}}Rest.random().build();
+const graphql{{modelName}} = {{modelName}}Graphql.random().build();
+
+// TODO: Write draft examples if any
+
+// TODO: Write preset examples if any
+// Example
+// const rest{{modelName}}Empty = {{modelName}}Rest.presets.empty().build();
+// const graphql{{modelName}}Empty = {{modelName}}Graphql.presets.empty().build();
```

Comment:
@CarlosCortizasCT are you planning to add these TODO's to this PR or a separate one, or these are just leftovers?

Reply:
Thanks for the feedback, Levente 👍 

My idea is that this to keep them. They are part of the TODOs the bootstrapped code will contain to give developers hints about what they should do next.